packages/fx-core/src/plugins/resource/bot/configs/scaffoldConfig.ts

```diff
@@ -35,11 +37,17 @@ export class ScaffoldConfig {
     ) {
       this.programmingLanguage = rawProgrammingLanguage as ProgrammingLanguage;
     }
+
+    const rawHostType = context.projectSettings?.pluginSettings?.[PluginBot.HOST_TYPE];
```

Comment:
`pluginSettings` is for all plugins, so make this host type for bot plugin only. `projectSettings.pluginSettings.fx-resource-bot.host-type`

Reply:
sorry, it has been fixed.